src/lib/apidoc.json

```diff
@@ -10,7 +10,7 @@
   },
   "servers": [
     {
-      "url": "https://lookup.ld4l.org/authorities",
+      "url": "https://lookup-stg.ld4l.org/authorities",

```

Comment:
Should this be reverted back to the production QA URI?

Reply:
Umm, definitely.